src/commands/force/source/deploy.ts

```diff
@@ -225,37 +225,79 @@ export class Deploy extends DeployCommand {
     return this.isAsync ? this.report(validatedDeployId) : this.poll(validatedDeployId);
   }
 
-  private progress(deploy: MetadataApiDeploy): void {
+  // Prints the current status of the deployment
+  private printStatus(data: MetadataApiDeployStatus): void {
+    if (!data.done) {
+      this.ux.log('');
+      this.ux.styledHeader(chalk.yellow('Status'));
+    } else {
+      if (data.completedDate) {
+        const deployStart: number = new Date(data.createdDate).getTime();
+        const deployEnd: number = new Date(data.completedDate).getTime();
+        this.ux.log(`Deployment finished in ${deployEnd - deployStart}ms`);
+      }
+      this.ux.log('');
+      const successHeader: string = chalk.green('Result');
+      const failureHeader: string = chalk.red('Result');
+      const header: string = data.success ? successHeader : failureHeader;
```

Comment:
Nit: could you we simplify this to `const header: string = data.success ? chalk.green('Result') : chalk.red('Result');`